pkg/execution/state/redis_state/queue.go

```diff
@@ -39,9 +40,10 @@ import (
 )
 
 const (
-	PartitionSelectionMax = int64(100)
-	PartitionPeekMax      = PartitionSelectionMax * 3
-	AccountPeekMax        = int64(30)
+	PartitionSelectionMax          = int64(100)
+	PartitionPeekMax               = PartitionSelectionMax * 3
+	AccountPeekMax                 = int64(30)
+	ShadowPartitionPeekMaxBacklogs = int64(100)
```

Comment:
this should be overridable, like `peekMax` in the queue struct

Reply:
not sure if there are benefits of doing EWMA atm, so we can defer the decision for now